stdlib/sampledata/sampledata.flux

```diff
@@ -0,0 +1,193 @@
+// Package sampledata provides functions that return basic sample datasets.
+package sampledata
+
+
+import "csv"
+
+// start and stop constants provide start and stop times to be used with `range`
+// when `_start` and `_stop` columns are required to demonstrate a transformation.
+//
+start = 2021-01-01T00:00:00Z
```

Comment:
Does this mean that all sample  data is within the range of start and stop?

Reply:
Yes, they do. I updated the comment to clarify.